exercises/clock/src/example/kotlin/Clock.kt

```diff
@@ -0,0 +1,46 @@
+class Clock(private var hours: Int, private var minutes: Int) {
+
+    companion object {
+        private val MINUTES_IN_AN_HOUR = 60
+        private val HOURS_IN_A_DAY = 24
+    }
+
+    init {
+        sanitiseTime()
+    }
+
+    override fun equals(other: Any?): Boolean {
```

Comment:
If you make this a `data class` you don't need to override the equals operator, it is already taken care of for you as well as equals/hashcode/toString (though you will still want to override the default toString for this case).